base-theme/layouts/partials/footer.html

```diff
@@ -1,2 +1,62 @@
-<footer>
+<footer id="footer-container" class="column-below-medium">
+    <section class="columns">
+      <div class="image-container">
+        <a id="mit-logo" href="https://www.mit.edu/" target="_blank">
+          <img src="/images/mit-logo-red.svg">
+        </a>
+        <a id="open-learning-logo"  href="https://openlearning.mit.edu/" target="_blank" class="pl-5">
+          <img src= "/images/mit-ol.png">
+        </a>
+      </div>
+      <div class="about-courseware">
+        <h5>
+          ABOUT MIT OPENCOURSEWARE
+        </h5>
+        <p>
+          MIT OpenCourseWare is an online publication of materials from over 2,500
+          MIT courses, freely sharing knowledge with learners and educators
+          around the world. <a href="/about">Learn more</a> »
+        </p>
+      </div>
+      <div class="large-and-above-only">
+        <h5>PROUD MEMBER OF : <a href="https://www.oeglobal.org/" target="_blank"><img class="oeg-logo" src="/images/oeg_logo.gif"></a></h5>
+      </div>
+    </section>
+    <section class="columns">
+      <div class="pd-14">
+        <ul>
+          <li><a href="https://accessibility.mit.edu">ACCESSIBILITY</a></li>
+          <li><a href="https://creativecommons.org/licenses/by-nc-sa/4.0/">Creative Commons License</a></li>
+          <li><a href="https://ocw.mit.edu/terms/">Terms and Conditions</a></li>
```

Comment:
```suggestion
          <li><a href="pages/privacy-and-terms-of-use/">Terms and Conditions</a></li>
```

I'd like to use a page that managed by the CMS. I think this will be the correct URL.